system-tests/tests/src/test/java/org/eclipse/dataspaceconnector/system/tests/local/FileTransferIntegrationTest.java

```diff
@@ -34,6 +35,7 @@
 import static org.eclipse.dataspaceconnector.system.tests.utils.FileTransferSimulationUtils.PROVIDER_ASSET_NAME;
 import static org.eclipse.dataspaceconnector.system.tests.utils.GatlingUtils.runGatling;
 
+@EndToEndTest
```

Comment:
Now as this test run part of integration test pipeline so we need to move `Gatling reports` step from verify.yaml to integrationtest.yml.

otherwise gatling reports will not be available and also we see this warning on unit tests runs.

<img width="393" alt="image" src="https://user-images.githubusercontent.com/555114/161025386-c33f4108-5603-44cd-bba2-06f9ad80e03d.png">
https://github.com/agera-edc/DataSpaceConnector/actions/runs/2069837614

Reply:
This comment is only valid if we move these tests to integrationtest.yml. As one comment is open on this topic that where we want to keep these tests.

Reply:
Good catch thanks! I will move the step to publish the report to the integration tests because one of the reason to have this tags was to run the test in a different agent to parallelise the run.